high_card.rb

```diff
@@ -1,5 +1,21 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+
+
+ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K"]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+deck = []
+deck = ranks.product(suits)
+puts deck.shuffle
+
+=begin
+
+=end
+# print "Enter player name"
+# player = gets.chomp
+# # player = []
+# puts "Welcome #{player}! Let's play high card!"
+
```

Comment:
You'll need to be in some kind of loop here if you want to add multiple players. One way that you could approach this is using a hash to represent a player.  This is what a player would look like:

```
{name: "Bob", card: nil}
```

In this way you have access to both the persons name and their card.